db/schema.rb

```diff
@@ -190,7 +190,6 @@
     t.string "facebook_handler"
     t.string "youtube_handler"
     t.string "github_handler"
-    t.boolean "destacat", default: false
```

Comment:
This is a strange field (in catalan...) and a strange change because it is not related to the PR